spec/timezone.html

```diff
@@ -5,6 +5,65 @@
   <h1>Temporal.TimeZone Objects</h1>
   <p>A Temporal.TimeZone object is an immutable Object referencing a time zone.</p>
 
+  <emu-clause id="sec-time-zone-names">
+    <h1>Time Zone Names</h1>
+
+    <p>
+      An ECMAScript implementation must support a number of built-in time zones.
+      Implementations must support a built-in time zone named *"UTC"* as well as the host environment's current time zone (if different from UTC), and may support any number of other built-in time zones.
```

Comment:
I don't think it's possible to say '_must_ support the host environment's time zone'. If I set my system time zone to `Camlandia/Camville`, [`Date.LocalTZA`](https://tc39.es/ecma262/#sec-local-time-zone-adjustment) will return 0. How should temporal handle this differently from that?

I think, if interacting with the IANA zone list is recommended for / only required in 402, then the only time zone temporal can mandate support for is UTC.